src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2095,25 +2092,37 @@ def list_locations(cmd, sku, linux_workers_enabled=None):
     return client.list_geo_regions(full_sku, linux_workers_enabled)
 
 
-def _check_zip_deployment_status(deployment_status_url, authorization, timeout=None):
+def _check_zip_deployment_status(cmd, rg_name, name, deployment_status_url, authorization, timeout=None):
     import requests
     total_trials = (int(timeout) // 2) if timeout else 450
     num_trials = 0
     while num_trials < total_trials:
         time.sleep(2)
         response = requests.get(deployment_status_url, headers=authorization)
+        time.sleep(2)
         res_dict = response.json()
         num_trials = num_trials + 1
         if res_dict.get('status', 0) == 3:
-            raise CLIError("Zip deployment failed. {}".format(res_dict))
+            # Turn on diagnostic logs
+            config_diagnostics(cmd, resource_group_name=rg_name, name=name,
+                               application_logging=True, web_server_logging='filesystem',
+                               docker_container_logging='filesystem', detailed_error_messages=True)
+            logger.warning('Please run the command ')
```

Comment:
this warning text seems redundant.

Reply:
yeah no idea what i was thinking when i wrote this line of code :-/